tests/group_functions/group_of.h

```diff
@@ -306,47 +310,57 @@ void predicate_function_of_sub_group(sycl::queue& queue) {
         sycl::sub_group sub_group = item.get_sub_group();
         T size = sub_group.get_local_linear_range();
 
-        T local_var(item.get_global_linear_id() + 1);
+        // Use the sub-group local ID (plus 1) as a variable against which to
+        // test our predicates. Note that this has a well-defined set of values
+        // [1,2,...,N] where N is the sub-group size. Note that the sub-group
+        // could also just be of size 1.
+        T local_var(sub_group.get_local_linear_id() + 1);
 
         // predicates
+        // The variable is never 0 for any member of the sub-group
```

Comment:
```suggestion
        // The variable is never 1 for any member of the sub-group
```
or
```suggestion
        // The variable is always 0 for any member of the sub-group
```
?

Reply:
I'll go with the former as it better matches the predicate's name. Thanks!